schema

```diff
@@ -1 +1 @@
-Subproject commit c4b69c3a5076d31e2d40d5ab7f053aab6bdbe938
```

Comment:
It happens when switching between branches with different json schema commits. I've been doing this to fix it (assuming you've run `git submodule init; git submodule update` once before):

```rm -rf schema
git submodule init
git submodule update```